test/addons/external-snapshotter/controller/kustomization.yaml

```diff
@@ -3,3 +3,15 @@ resources:
   - https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/release-8.2/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml
   - https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/release-8.2/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml
 namespace: kube-system
+patches:
+  # The enable-volumegroup-snapshot flag has been replaced by a feature-gates
+  # flag https://github.com/kubernetes-csi/external-snapshotter/pull/1194. The
+  # feature gate is disabled by default. To enable VolumeGroupSnapshot, adding
+  # --feature-gates=CSIVolumeGroupSnapshot=true in snapshot-controller.
```

Comment:
The history of enable-volumegroup-snapshot is not very interesting now, and wed on't need to mention snapshot-controller since this is a patch of the snapshot controller.

I would just write here this to explain why the patch exists.

    # Enable volume group replication support